R/Driver.R

```diff
@@ -75,10 +75,10 @@ setMethod(
 #' @param ... Additional ODBC keywords. These will be joined with the other
 #'   arguments to form the final connection string.
 #'
-#'   Note that all generic ODBC parameter names, and the values of `drv` and
-#'   `dsn`, are case-insensitive. The case-sensitivity of the values of generic
-#'   arguments, as well as the names and values of driver-specific arguments,
-#'   are determined by the driver `drv`.
+#'   Note that ODBC parameter names are case-insensitive so that (e.g.) `DRV`
+#'   and `drv` are equivalent. Since this is different to R and a possible
+#'   source of confusion, odbc will error if you supply multiple arguments that
+#'   have the same name when case is ignored.
```

Comment:
I think that might be true, but I have yet to see a driver where that's the case.  

Additionally, even if there is a driver that is case sensitive, the error isn't going to cause problems because I don't think any driver would be silly enough to have two different arguments that have the same name when you ignore case.